reviewed.md

```diff
@@ -0,0 +1,31 @@
+---
+# A human-friendly name of your listing
+title: reviewed
+# A short description of what your app does
+description: **reviewed** is a Github App built with [probot](https://github.com/probot/probot) that automatically add label to pull request when all of reviewers approved.
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: reviewed
+# Include a few screenshots that show your app in action
+screenshots: 
+- https://github.com/g-ozawa/reviewed/raw/master/assets/reviewed_ss.png
+# The GitHub usernames of anyone who authored the app
+authors: [ g-ozawa ]
+# The repository where the code is located
+repository: g-ozawa/reviewed
+# The address where this app is deployed
+host: https://reviewed.now.sh
+---
+
+# reviewed
+
+**reviewed** is a Github App built with [probot](https://github.com/probot/probot) that automatically add label to pull request when all of reviewers approved.
```

Comment:
Again rm the Probot/GitHub Apps bit, everything on the website is a GitHub App built with Probot.